CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ This guideline intends to make contribtuions to Caliper easier by:
 
 If you have further suggestions about improving the guideline, then you can follow the presented workflow to make your contribution.
 
+Jump to [Contributing documentation](./docs/Contributing.md) to get started on your journey.
+
```

Comment:
For simplicity this should point to the served docs and specify the 0.6.0 location here.